docs/Inputs.md

```diff
@@ -117,6 +196,49 @@ You can customize the rich text editor toolbar using the `toolbar` attribute, as
 <RichTextInput source="body" toolbar={[ ['bold', 'italic', 'underline', 'link'] ]} />
 ```
 
+## `<SelectInput>`
+
+To let users choose a value in a list using a dropdown, use `<SelectInput>`. It renders using [Material ui's `<SelectField>`](http://www.material-ui.com/#/components/select-field). Set the `choices` attribute to determine the options (with `id`, `name` tuples):
+
+```js
+import { Edit, SelectInput } from 'admin-on-rest/mui';
+
+export const PostEdit = (props) => (
+    <Edit {...props}>
+        <SelectInput source="category" choices={[
+            { id: 'programming', name: 'Programming' },
+            { id: 'lifestyle', name: 'Lifestyle' },
+            { id: 'photography', name: 'Photography' },
+        ]} />
+    </Edit>
+);
+```
+
+![SelectInput](./img/select-input.gif)
+
+You can also customize the properties to use for the option name and value, thanks to the `optionText` and `optionValue` attributes:
+
+```js
+<SelectInput label="Author" source="author_id" optionText="full_name" optionValue="_id" choices={[
+    { _id: 123, full_name: 'Leo Tolstoi', sex: 'M' },
+    { _id: 456, full_name: 'Jane Austen', sex: 'F' },
+]} />
+```
+
+Lastly, use the `options` attribute if you want to override any of Material UI's `<SelectField>` attributes:
+
+{% raw %}
+```js
+<SelectInput source="category" options={{
```

Comment:
Not related to this PR, but an `options` attribute on a a select might be confused with the select options elements